dist/index.d.ts

```diff
@@ -1,13 +1,14 @@
-enum APIVersion {
+declare enum APIVersion {
```

Comment:
Yeah because we use dynamic handling of the schema as of now and roll up generates the relevant esm files for the module. So for typescript we could add, but that would mean every method we change we'll have to update at two places. 

If you want I can do that way.